packages/database/src/index.standalone.ts

```diff
@@ -0,0 +1,88 @@
+import { Client } from 'faye-websocket';
```

Comment:
Add license header and remove extra whitespace at the end.

I don't quite understand the purpose of this file. Isn't `database-compat/src/index.standalone.ts` enough?

Reply:
Can we have a shared list of exports like in the other clients (`api.ts`)? These lists go out of sync really fast.

Reply:
`database-compat/src/index.standalone.ts` needs to import from an entry point from `@firebase/database` which doesn't depend on `@firebase/app`, otherwise you still depend on `@firebase/app` transitively.

@schmidt-sebastian Done. Added `api.standalone.ts` which `api.ts` re-exports.